bundles/admin/css.rst

```diff
@@ -0,0 +1,15 @@
+Include custom css
+==================
+
+Sulu provides the package require-css_ to manage custom css. you can load
+css files in any require component with the prefix `css!`. You can omit the
+file extension.
+
+.. code-block:: javascript
+
+    define(['css!suluadmin/css/main'], function(){});
```

Comment:
Don't you want to add a more realistic entry? I thought you had to add a path like `suluadmincss`